src/buildHTML.js

```diff
@@ -632,6 +635,12 @@ groupTypes.array = function(group, options) {
                 gap = 0;
             }
         }
+        // In AMS multiline environments such as aligned and gathered, rows
+        // correspond to lines that have additional \jot added to the
+        // \baselineskip via \openup.
+        if (group.value.rowsAreLines) {
```

Comment:
Agreed, that is confusing.  Renamed to `addJot`, which seems clear enough (as `\jot` is an interline spacing).